### PR TITLE
feat: add footer component

### PR DESCRIPTION
**Pull request by amt-augustineopokukesse**
Added footer component
Refactored footer feature to be reusable

### CODE REVIEW

File: apps/iglidur-designer/src/app/libs/header/src/lib/footer/footer.component.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  imports: [
+    CommonModule,
+    MatFormFieldModule,
+    MatIconModule,
+    MatInputModule,
+    FormsModule,
+    MatButtonModule,
+  ],
+  templateUrl: './footer.component.html',
+  encapsulation: ViewEncapsulation.None,
```
Comment (Bilal-Abubakari) [inline apps/iglidur-designer/src/app/libs/header/src/lib/footer/footer.component.ts:21]: Why this?

Comment (amt-augustineopokukesse) [reply apps/iglidur-designer/src/app/libs/header/src/lib/footer/footer.component.ts:21]: I wasn't able to style the Angular material elements I was using, so I found this solution that enabled me to do so.
I have not fully understood the use of the custom theming but I am working on it.

Comment (Bilal-Abubakari) [reply apps/iglidur-designer/src/app/libs/header/src/lib/footer/footer.component.ts:21]: Please this solution is not right. Encapsulation should be emulated

[the remark this answers is not in the document]
Comment (amt-augustineopokukesse) [reply apps/iglidur-designer/src/app/libs/header/src/lib/footer/footer.component.ts:21]: Sure.. will do that

Comment (amt-augustineopokukesse) [reply apps/iglidur-designer/src/app/libs/header/src/lib/footer/footer.component.ts:21]: Okay please.. will work on it

File: apps/iglidur-designer/src/app/libs/header/src/lib/footer/footer.component.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    MatFormFieldModule,
+    MatIconModule,
+    MatInputModule,
+    FormsModule,
+    MatButtonModule,
+    TranslateModule,
+  ],
+  templateUrl: './footer.component.html',
+  styleUrl: './footer.component.scss',
+})
```
Comment (Bilal-Abubakari) [inline apps/iglidur-designer/src/app/libs/header/src/lib/footer/footer.component.ts:26]: change changeDetection to onPush

**Review by Bilal-Abubakari — APPROVED** [review]

**Pull request merged**